src/test/cli-test.ts

```diff
@@ -26,8 +28,9 @@ describe('cli', () => {
     // tslint:disable-next-line:no-let
     let info = ''
     const report: cli.Report = {
-      error: s => (error += s),
-      info: s => (info += s),
+      logError: s => (error += s),
+      logResult: s => (error += s),
```

Comment:
result += s?

Reply:
It's ok. because actually we append error and result to the same file.